assets/src/edit-story/components/movable/moveStyle.js

```diff
@@ -68,4 +68,9 @@ export const GlobalStyle = createGlobalStyle`
 		top: -16px;
 		height: 16px;
 	}
+
+	.default-movable.dragging .moveable-line.moveable-rotation-line,
```

Comment:
What are the benefits of doing this vs simply saying `resizable={false} rotatable={false}` on dragging?

Reply:
We already do that, it removes the handles (corners) but not the box. This disables the outline box while dragging. Sam thinks it's redundant to have the box since you already know what you're dragging.

Reply:
I see.